XamarinCommunityToolkitSample/ViewModels/Behaviors/BehaviorsGalleryViewModel.cs

```diff
@@ -1,15 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using XamarinCommunityToolkitSample.Models.Behaviors;
+﻿using System.Collections.Generic;
+using Xamarin.Forms;
+using XamarinCommunityToolkitSample.Models;
+using XamarinCommunityToolkitSample.Pages.Behaviors;
 
 namespace XamarinCommunityToolkitSample.ViewModels.Behaviors
 {
     public class BehaviorsGalleryViewModel : BaseViewModel
     {
-        public IEnumerable<BehaviorSectionModel> Items { get; } =
-            ((BehaviorSectionId[])Enum
-            .GetValues(typeof(BehaviorSectionId)))
-            .Select(id => new BehaviorSectionModel(id));
+        public IEnumerable<SectionModel> Items { get; } = new List<SectionModel> {
+            new SectionModel(
+                typeof(EmailValidationBehaviorPage),
+                "EmailValidationBehavior",
+                "Changes an Entry's text color when an invalid e-mail address is provided."
+            ),
+            new SectionModel(
+                typeof(EventToCommandBehaviorPage),
+                "EventToCommandBehavior",
+                "Turns any event into a command that can be bound to."
+            ),
+            new SectionModel(
+                typeof(NumericValidationBehaviorPage),
+                "NumericValidationBehavior",
+                "Changes an Entry's text color when an invalid numeric value is provided."
```

Comment:
_(Not something that have to be changed now)_
Are we planning to add localization to the app? Maybe we can have those strings in a Resource file, so if we want to support different languages later is easier to add.

Reply:
That's a great idea! We might borrow some inspiration from GitTrends

Reply:
I guess we should probably open a new issue for it.

Reply:
Yes! Please do so we can track it better